GliaWidgetsTests/Coordinator/RootCoordinator.Environment.Failing.swift

```diff
@@ -89,12 +89,12 @@ extension EngagementCoordinator.Environment {
         proximityManager: .failing,
         log: .failing,
         snackBar: .failing,
-        operatorRequestHandlerService: .failing,
         maximumUploads: {
             fail("\(Self.self).maximumUploads")
             return 2
         },
         cameraDeviceManager: { .failing },
-        flipCameraButtonStyle: .nop
+        flipCameraButtonStyle: .nop,
+        alertManager: .mock()
```

Comment:
I'd recommend to create `failing` instance

Reply:
added [c08f255](https://github.com/salemove/ios-sdk-widgets/pull/968/commits/c08f255d539b691b212f88c3695b2f272a154aad)